src/test/scala/org/broadinstitute/dsde/firecloud/dataaccess/MockSearchDAO.scala

```diff
@@ -42,4 +43,8 @@ class MockSearchDAO extends SearchDAO {
     Future(LibrarySearchResponse(librarySearchParams, 0, Seq[JsValue](), Seq[LibraryAggregationResponse]()))
   }
 
+  override def fieldSuggest(field: String, text: String): Future[Seq[String]] = {
```

Comment:
this is tested by the last test in LibraryAPIServiceSpec, the call to populate suggest invokes the mock dao